xline-client/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "xline-client"
+version = "0.1.0"
+edition = "2021"
+authors = ["DatenLord <dev@datenlord.io>"]
+description = "Client for Xline"
+license = "Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/datenlord/Xline/tree/master/xline-client"
+categories = ["Client"]
```

Comment:
How about Command Line Tool here?

Reply:
This crate is just the client API. The binary command line tool will be implemented in another crate.